adapters/rest-adapter-generator/src/main/java/uk/gov/justice/services/adapters/rest/generator/Generators.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.services.adapters.rest.generator;
+
+import static uk.gov.justice.services.adapters.rest.generator.Names.baseUriPathWithoutContext;
+
+import uk.gov.justice.services.core.annotation.Component;
+
+import java.util.Comparator;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.raml.model.MimeType;
+import org.raml.model.Raml;
+
+final class Generators {
```

Comment:
This was intentional, I wanted to keep the utils classes package local unless otherwise needed.  We do need to be consistent though so I will make sure the other utils classes are set the same way.